src/htm/algorithms/SpatialPooler.hpp

```diff
@@ -202,6 +214,7 @@ class SpatialPooler : public Serializable
   @param wrapAround boolean value that determines whether or not inputs
         at the beginning and end of an input dimension are considered
         neighbors for the purpose of mapping inputs to columns.
+	//TODO does it hurt to set this to always true? We could rm NonWrappingNeighbourhood
```

Comment:
> whether or not inputs
        at the beginning and end of an input dimension are considered
        neighbors for the purpose of mapping inputs to columns

biologically, if we assume hierarchy, a Region we model with SP is a portion ("rectangle") on a 2D sheet. Its input field is another 2D sheet (or retina, ...) -> so inputs on one side are not close to the others. So we should leave this OFF?